### PR TITLE
Coding style - com_newsfeeds

### PR DESCRIPTION
**Pull request by marcochirienti**
### Summary of Changes

Added one white-space before the PHP code closing tag where missing.

### Testing Instructions

Code review.

### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/components/com_newsfeeds/views/newsfeeds/tmpl/default.php [hunk 1]
```diff
@@ -145,7 +145,7 @@
 										<?php echo $this->escape($item->name); ?>
 								<?php endif; ?>
 								<span class="small">
-									<?php echo JText::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->alias));?>
+									<?php echo JText::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->alias)) ;?>
```
Comment (shur) [inline administrator/components/com_newsfeeds/views/newsfeeds/tmpl/default.php]: wrong place for space

**Review by shur — CHANGES_REQUESTED** [review]

**Comment by marcochirienti** [comment]
Fixed. Thank you @shur .

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on bbee8884ee22a7c455ed4c2a019330961c69cb18<br /><br />by code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13139">issues.joomla.org/tracker/joomla-cms/13139</a>.</sub>

**Review by zero-24 — APPROVED** [review]
Looks ok from review

**Pull request merged**